contracts/contracts/ramps/paylah/PaylahRamp.sol

```diff
@@ -78,7 +78,9 @@ contract PaylahRamp is Ownable {
 
     struct Deposit {
         address depositor;
-        uint256[8] upiId;
+        bytes32 offRampIdHash;              // Identifying hash used for off-ramping, hash(alias+4 mobile digits) different
```

Comment:
Let's store this raw, alias and 4-digit mobile number. You can think of it as storing UPI ID (sachin3929@paytm) but in two separate fields, alias (sachin) and mobile num (3929). Then introduce back the `_getUpiIdHash` function you deleted at the bottom to calculate their hash. Also in the offRamp function, let's take the input alias + 4-digit mobile number as two separate values and calculate their hash.

Reply:
Why?

Reply:
It feels unnecessary, we did that with UPI because that raw value is what is used to pay and then is present in the e-mail. Here the `offRampIdHash` isn't used to pay so we don't need the information we just have to be able to match it to the output form the e-mail. Would lean towards, and think our users would prefer, not putting that stuff on-chain if not necessary.